examples/permissioned-voting/scripts/vote/result.js

```diff
@@ -26,24 +26,24 @@ async function run (runtimeEnv, deployer) {
     console.log('The Winner is CandidateA!!');
   }
 
-  // const txnParam = {
-  //   type: types.TransactionType.DeleteApp,
-  //   sign: types.SignType.SecretKey,
-  //   fromAccount: votingAdminAccount,
-  //   appID: appInfo.appID,
-  //   payFlags: {}
-  // };
-
-  // // Delete Application
-  // console.log('Deleting Application');
-  // await executeTransaction(deployer, txnParam);
-
-  // txnParam.fromAccount = alice;
-  // txnParam.type = types.TransactionType.ClearApp;
-
-  // // Clear voter's account
-  // console.log("Clearing Alice's Account");
-  // await executeTransaction(deployer, txnParam);
```

Comment:
Why was code commented before?

Reply:
i don't know

Reply:
It seams that @amityadav0 commented it out last week. Maybe some mistake?

Reply:
yes